CHANGELOG.md

```diff
@@ -41,25 +45,6 @@ release.
 <a href="doc/changelogs/CHANGELOG_V8.md#8.1.0">8.1.0</a><br/>
 <a href="doc/changelogs/CHANGELOG_V8.md#8.0.0">8.0.0</a><br/>
     </td>
-    <td valign="top">
-<b><a href="doc/changelogs/CHANGELOG_V7.md#7.10.1">7.10.1</a></b><br/>
```

Comment:
Why are we removing 7?

Reply:
For the same reason 5 is not listed. The changelogs are still there and accessible